pkg/webui/console/containers/sidebar/sidebar.styl

```diff
@@ -24,9 +24,6 @@
   left: 0
   height: 100vh
   sidebar-transition(left)
-  // Stylus has trouble adding multiple properties to a function
-  // so we have to overwrite it like this.
-  transition-property: left, width
```

Comment:
This will also remove the width-transition which occurs when the sidebar moves from the wide to the narrow layout. (visible at 1600px width)

Reply:
I changed the property to be included only for screens bigger than xl so we don't have the weird sliding behaviour on smaller screen when the sidebar is hidden.